scripts/ts-utils/setup-enclave.ts

```diff
@@ -34,11 +35,39 @@ async function transfer(api: any, Alice: any) {
             });
     });
 }
-async function updateScheduledEnclave(api: any, signAccount: any) {
+async function setTeerexAdmin(api: any, Alice: any) {
     return new Promise(async (resolve, reject) => {
         await api.tx.sudo
-            .sudo(api.tx.teerex.updateScheduledEnclave(0, hexToU8a(`0x${mrenclave}`)))
-            .signAndSend(signAccount, ({ status, events, dispatchError }) => {
+            .sudo(api.tx.teerex.setAdmin("esqZdrqhgH8zy1wqYh1aLKoRyoRWLFbX9M62eKfaTAoK67pJ5"))
```

Comment:
One small suggestion is to avoid using hardcoded values in script-type code. Instead, consider either reading from a configuration file or accepting input from the command line.:p

Reply:
Yes - or at least having a comment about where this address comes from :)

Reply:
Added a comment about this address ✅